<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UreticiProfil</class>
 <widget class="QWidget" name="UreticiProfil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>2000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Üretici Profil Ekranı</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>621</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 35pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>ÜRÜNLERİM:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_geri_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>1730</y>
     <width>311</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Geri</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>550</y>
     <width>771</width>
     <height>1041</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCommandLinkButton" name="LinkButton_3">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="LinkButton_1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="LinkButton_5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="LinkButton_4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="LinkButton_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>60</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 25pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Yeni Ürün </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>150</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 70pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>1670</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 50pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
